package.json

```diff
@@ -28,11 +28,15 @@
   ],
   "jest": {
     "collectCoverage": true,
-    "moduleFileExtensions": [
-      "js"
-    ],
-    "moduleDirectories": [
-      "node_modules"
+	  "coverageDirectory": "./coverage/",
```

Comment:
Are most of these not `defaults` anyways? Also the question if we should add this to `webpack-defaults` then